### PR TITLE
fix: Fix for analytics not getting called.

### PR DESCRIPTION
**Pull request by schinkowski**

### CODE REVIEW

**Review by seandillon92 — APPROVED** [review]

File: Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs [hunk 1]
```diff
@@ -71,11 +71,12 @@ internal abstract class AnalyticsDataHandlerBase {
             return true;
         }
 
-        [Conditional("DEBUG_ANALYTICS")]
         private static void LogIfWarning(AnalyticsResult resp) {
+#if DEBUG_ANALYTICS
```
Comment (sindharta) [inline Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs:75]: I thought this had the same meaning.
What is the difference ?

Comment (schinkowski) [reply Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs:75]: The difference is that with the conditional compile, the method to send the analytics never gets called without the DEBUG_ANALYTICS define.

Comment (sindharta) [reply Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs:75]: Ah, the whole execution gets culled when it's called like this
```
            LogIfWarning(
            EditorAnalytics.SendEventWithLimit(
                eventName,
                data,
                version));
```

Got it.

**Review by sindharta — APPROVED** [review]

File: Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs (hunk 1 above)
Comment (schinkowski) [reply Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs:75]: Yeah exactly

**Review by SonnyCampbellUnity — APPROVED** [review]

**Review by sindharta-tanuwijaya — APPROVED** [review]

**Pull request merged**